packages/mux-uploader/src/mux-uploader.ts

```diff
@@ -135,6 +134,28 @@ class MuxUploaderElement extends globalThis.HTMLElement implements MuxUploaderEl
     this._endpoint = value;
   }
 
+  get noDrop(): boolean {
+    return this.hasAttribute('noDrop');
+  }
+
+  set noDrop(value: boolean) {
+    if (value) {
+      this.setAttribute('noDrop', '');
+    } else {
+      this.removeAttribute('noDrop');
+    }
```

Comment:
```suggestion
    this.toggleAttribute('nodrop', Boolean(value));
```

we can avoid if / else and use lowercase attribute naming